source/Xunit.Gherkin.Quick/StepDefinition/GivenWhenThen.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Xunit.Gherkin.Quick
+{
+
+    public sealed class GivenWhenThenAttribute : StepDefinitionAttributeBase
```

Comment:
What is this class for? I think you accidentally renamed ButAttribute into this? not sure what's the intent here. Please clarify.